HACKING.rst.txt

```diff
@@ -128,13 +139,50 @@ deprecation message for the changed behavior.  This should be part of the
 review process.
 
 
+Reviewing pull requests
+=======================
+
+Reviewing open pull requests (PRs) is very welcome, and a valuable way to help
+increase the speed at which the project moves forward.  We encourage everyone
+to get involved in the review process; it's also a great way to get familiar
+with the code base.  Reviewers should ask themselves some or all of the
+following questions:
+
+- Was this change adequately discussed (relevant for new features and changes
+  in existing behavior)?
+- Is the feature scientifically sound? Algorithms may be known to work based on
+  literature; otherwise, closer look at correctness is valuable.
+- Is the intended behavior clear under all conditions (e.g. unexpected inputs
+  like empty arrays or nan/inf values)?
+- Does the code meet the quality, test and documentation expectation outline
+  under `Contributing new code`_?
+
+If you have specific knowledge/experience in a particular
+area (say "optimization algorithms" or "special functions") then reviewing PRs
+in that area is especially valuable - sometimes PRs with technical code have to
+wait for a long time to get merged due to a shortage of appropriate reviewers.
+
+
 Other ways to contribute
 ========================
 
-There are many ways to contribute other than contributing code.  Participating
-in discussions on the scipy-user and scipy-dev *mailing lists* is a contribution
-in itself.  The `scipy.org`_ *website* contains a lot of information on the
-SciPy community and can always use a new pair of hands.
+There are many ways to contribute other than contributing code.
+
+Triaging issues (investigating bug reports for validity and possible actions to
+take) is also a useful activity.  SciPy has many hundreds of open issues;
+closing invalid ones and correctly labeling valid ones (ideally with some first
+thoughts in a comment) allows prioritizing maintenance work and finding related
+issues easily when working on an existing function or submodule.
+
+Participating in discussions on the scipy-user and scipy-dev *mailing lists* is
```

Comment:
Made links for ``mailing lists``. Didn't take the first suggestion because for half the entries in that list it's hard to make it links without making the whole thing very messy with labels (there's no 1:1 correspondence between sections in this doc and this list).